inst/markdown/nashConvergence-qttyTarget.Rmd

```diff
@@ -0,0 +1,327 @@
+---
+title: "Regional quantity target convergence"
+output:
+  html_document:
+    toc: false
+    toc_float: true
+    toc_depth: 5
+    code_folding: hide
+params:
+  gdx: "fulldata.gdx"
+  warning: false
+  message: false
+  figWidth: 16
+---
+
+<style>
+  #header {
+    display: none !important
+  }
+  .main-container {
+    max-width: 95% !important;
+  }
+  .toc-content {
+    padding-left: 0px !important;
+  }
+  .svg-container {
+    margin: auto !important;
+  }
+  .code-folding-btn {
+    display: none;
+  }
+</style>
+
+<script type="text/javascript">
+$(document).ready(function() {
+  var $tocItems = $(".tocify-subheader li.tocify-item"); // selector for all TOC items
+  var $tabs     = $("a[role=\"tab\"]");                  // selector for all tabs
+  var $panels   = $("div[role=\"tabpanel\"]");           // selector for all tabpanels
+
+  $tocItems.hide();  // hide all TOC items
+
+  // get the tab name for each section header (e.g. mammals)
+  // and assign it to its corresponding toc item
+  $panels.find("div[data-unique]").each(function() {
+    var key = $(this).attr("data-unique");
+    var tab = $(this).closest("div[role=\"tabpanel\"]").attr("id");
+    $tocItems.filter("[data-unique=\"" + key + "\"]").attr("tab", tab)
+  })
+  // now each toc item knows to which tab panel it is pointing
+
+  // show the toc items that point to sections on the first panel
+  $tocItems.filter(function() {
+    return($(this).attr("tab") === $tabs.first().text());
+  }).toggle();
+
+  // assign an onclick event to the tabs..
+  $tabs.on("click", function() {
+    $tocItems.hide();  // ... hide all TOC items
+    var key = $(this).text(); // ... get the name of the tab clicked
+    $tocItems.filter(function() {  // ... filter for the toc items pointing to that tab
+      return($(this).attr("tab") === key);
+    }).toggle();  // ... and show them
+  });  
+});
+</script>
+
+```{r loading_libraries, include=FALSE}
```

Comment:
Please consider applying auto-formatting and auto-indentation  (there are helpers in RStudio) and removing commented out code.

Reply:
https://github.com/pik-piam/remind2/pull/741/commits/2cfcfe05bef14630fcec4668ec9dd03bb5b0d593